layers/meta-resin-raspberrypi/conf/samples/local.conf.sample

```diff
@@ -62,3 +62,5 @@ BB_DISKMON_DIRS = "\
     ABORT,${SSTATE_DIR},100M,1K"
 
 CONF_VERSION = "1"
+
+HOSTTOOLS += "docker"
```

Comment:
Can you point to some docs on the usage of this var? Or maybe detail on it in the git log?

Reply:
http://www.yoctoproject.org/docs/current/ref-manual/ref-manual.html#var-HOSTTOOLS

It looks like Pyro is strict about what host binaries are allowed to be called from recipes.